content/geometry/human-structure/Grp_Equipment_Dynamic_(T).adoc

```diff
@@ -0,0 +1,28 @@
+= Grp_Equipment_Dynamic (T)
+
+Add a description
+
+.<Table title>
+[%header, cols="20, 80"]
+|===
+
+2+^| <Header>
+
+| *Origin*
+| ...
+
+| *x-axis*
+| ...
+
+| *y-axis*
+| ...
+
+| *z-axis*
+| ...
+|===
+
+
+Add a figure.
+
+.<Figure caption>
+//image::images/Vehicle_Structure_Door_Coord_Frame.svg[width=70%, scalewidth=10cm]
```

Comment:
```suggestion
//image::images/Vehicle_Structure_Door_Coord_Frame.svg[width=70%, scalewidth=10cm]

////
If equipment is very specific for a human asset, rather small, not self-contained or animated, it can be part of the human asset
////
```